transforms/angle-brackets/transform.js

```diff
@@ -435,16 +437,23 @@ function transformToAngleBracket(fileInfo, config, invokableData) {
         node.loc.source !== '(synthetic)' &&
         !shouldIgnoreMustacheStatement(tagName, config, invokableData);
       const isNestedComponent = isNestedComponentTagName(tagName);
+      const isNotBlockParamValue =
+        !havingBlockParams && !currentBlockParams.includes(node.path.original);
 
       if (
         isValidMustache &&
-        (node.hash.pairs.length > 0 || node.params.length > 0 || isNestedComponent)
+        (node.hash.pairs.length > 0 || node.params.length > 0 || isNestedComponent) &&
+        isNotBlockParamValue
       ) {
         return transformNode(node, fileInfo, config);
       }
     },
     BlockStatement(node) {
       let tagName = `${node.path.original}`;
+      havingBlockParams = node.program.blockParams.length > 0;
+      if (havingBlockParams) {
+        currentBlockParams = node.program.blockParams;
```

Comment:
this doesn't work for nested block params

Reply:
@Turbo87 Can you give me an example?

Reply:
I think the corresponding template-lint rule has something like this implemented, or you can have a look at the implicit-this codemod